### PR TITLE
code base is refactored 

### LINKED ISSUE #3: Code base is refactored
Single file with code MUST be splitted into framework-/utils-/server-related modules. Use your imagination to do the best code split and dependencies untying

### PR DESCRIPTION
**Pull request by Anastasiya-Taranova**
closes #3

### CODE REVIEW

File: src/framework/utils/logging.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+def mute_root_logger():
+    root_logger = logging.getLogger()
+    root_logger.setLevel(logging.CRITICAL)
+    for _handler in root_logger.handlers:
+        root_logger.removeHandler(_handler)
+
+
+def configure_logging(logger_name: str) -> logging.Logger:
+    debug = 1
```
Comment (tgrx) [inline src/framework/utils/logging.py:12]: reuse from dynaconf

File: src/framework/supported_api/blog/validate_api.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+from framework.supported_api.blog.schemas.posts import Post
+from framework.supported_api.blog.schemas.posts import PostApi
+from framework.supported_api.blog.schemas.posts import PostList
+from framework.supported_api.blog.schemas.posts import PostListApi
+from framework.supported_api.blog.schemas.users import User
+from framework.supported_api.blog.schemas.users import UserList
+from framework.supported_api.blog.schemas.users import UserListApi
+
+API_TIMEOUT = 2
```
Comment (tgrx) [inline src/framework/supported_api/blog/validate_api.py:17]: please place this under settings and make configurable via dynaconf

File: src/framework/supported_api/blog/validate_api.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    posts = get_all_posts(test_api_server)
+    validate_post_in_posts(new_post, posts)
+
+    existing_post = get_post_by_id(test_api_server, new_post.id)
+    validate_post(existing_post, new_post)
+
+
+def get_post_by_id(server: Text, new_post: int) -> Post:
+    url = f"{server}/api/v1/blog/post/{new_post}"
```
Comment (tgrx) [inline src/framework/supported_api/blog/validate_api.py:38]: please use common url

Comment (tgrx) [reply src/framework/supported_api/blog/validate_api.py:38]: please consider using a single function to make requests to the api, independent of particular schema

**Review by tgrx — CHANGES_REQUESTED** [review]

File: src/framework/utils/logging.py (hunk 1 above)
Comment (Anastasiya-Taranova) [reply src/framework/utils/logging.py:12]: +

File: src/framework/supported_api/blog/validate_api.py (hunk 2 above)
Comment (Anastasiya-Taranova) [reply src/framework/supported_api/blog/validate_api.py:17]: +++

File: src/framework/supported_api/blog/validate_api.py (hunk 3 above)
Comment (Anastasiya-Taranova) [reply src/framework/supported_api/blog/validate_api.py:50]: +++

File: src/main/main.py [hunk 4]
```diff
@@ -141,90 +52,5 @@ async def view_test(
...
-    assert response.status_code == 201, f"{url} does not work: {response.status_code}"
-    api_response_json = response.json()
-    api_response = PostApi.parse_obj(api_response_json)
-    post = api_response.data
-    return post
-
-
 if __name__ == "__main__":
-    uvicorn.run(app, host="127.0.0.1", port=7080)
+    uvicorn.run(app, host="127.0.0.1", port=7070)
```
Comment (tgrx) [inline src/main/main.py:56]: dynaconf please

Comment (tgrx) [reply src/main/main.py:56]: HOST must be `0.0.0.0`

**Review by tgrx — CHANGES_REQUESTED** [review]

**Review by tgrx — APPROVED** [review]

**Pull request merged**